index.js

```diff
@@ -107,15 +111,42 @@ class UpdateNotifier {
 			};
 		});
 	}
+	isYarn() {
+		const bin = this.packageBin;
+		let cliName = null;
+
+		switch (typeof bin) {
+			case 'string':
+				cliName = this.packageName;
+				break;
+			case 'object':
+				cliName = Object.keys(bin)[0];
+				break;
+			case 'undefined':
+			default:
+				return false;
+		}
+
+		try {
+			const realpath = fs.realpathSync(which().sync(cliName, {nothrow: true}));
+			if (realpath.match(/yarn\/global/)) {
```

Comment:
I could be wrong, but this literal RegExp doesn't look to be Windows-friendly (hardcoded forward slash).

Perhaps we could simplify this to:

```js
return realpath.includes(path.join('yarn', 'global'));
```

Reply:
Thanks, I will try it in Windows.